spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.feature 'ingredients index page' do
+  describe 'when visiting /ingredients' do
+    scenario 'US1 see a list of all ingredients with name and cost' do
+      cheese = Ingredient.create!(name: 'Cheese', cost: 1)
+      eggs = Ingredient.create!(name: 'Eggs', cost: 2)
+      bacon = Ingredient.create!(name: 'Bacon', cost: 3)
+
+      visit '/ingredients'
+
+      expect(page).to have_content(cheese.name)
+      expect(page).to have_content(cheese.cost)
+      expect(page).to have_content(eggs.name)
+      expect(page).to have_content(eggs.cost)
+      expect(page).to have_content(bacon.name)
+      expect(page).to have_content(bacon.cost)
```

Comment:
This test is good, but could be more robust with `within` blocks. For example, `expect(page).to have_content(bacon.cost)` is just expecting the number 3 to be *somewhere* on the page. What if there was a phone number on the page that contained a 3? That could give a false positive. Within blocks are handy so that you can narrow down where you want to see the 3 (i.e. in the section of the page just for bacon)